tibco/src/test/java/org/frankframework/extensions/tibco/pipes/ObfuscatePipeTest.java

```diff
@@ -63,28 +64,31 @@ void testEmptyInput() throws ConfigurationException, PipeStartException, IOExcep
 	}
 
 	@Test
-	@Disabled("This test is not always working somehow, but it works in production at customer site.")
-	void testLibraryDoesWork() throws AXSecurityException {
-		String result = ObfuscationEngine.encrypt(plainText.toCharArray());
-		char[] decrypted = ObfuscationEngine.decrypt(result);
+	void testObfuscationEngine() throws Exception {
+		String encrypted = ObfuscationEngine.encrypt(PLAIN_TEXT);
+
+		assertEquals(PLAIN_TEXT, ObfuscationEngine.decrypt(encrypted));
 
-		assertEquals(plainText, new String(decrypted));
+		// Test from the site we based the ObfuscationEngine off
+		String password = "Amadeus@123";
+		String encryptedPassword = ObfuscationEngine.encrypt(password);
+
+		assertEquals(password, ObfuscationEngine.decrypt(encryptedPassword));
 	}
 
 	@Test
-	@Disabled("This test is not always working somehow, but it works in production at customer site.")
 	void testDecryption() throws ConfigurationException, PipeStartException, IOException, PipeRunException {
 		// Arrange
 		pipe.setDirection(ObfuscatePipe.Direction.DEOBFUSCATE);
 		configureAndStartPipe();
-		byte[] inputString = "#!IE4R9xiIP+k4jLhPT1m4wubEcOfiMsq7eB6K93utkaqyrsMem/8uPWf7Ktq4JllwjsedkKsrMcYbmP0dvR5GzfnrrO1/MhVvPbWJLV0oxdI=".getBytes(StandardCharsets.UTF_8);
```

Comment:
Deze string was dus gewoon incorrect, dit resulteerde in een `javax.crypto.BadPaddingException`, zowel op java 8 als 21.